infra/data_collection/github/utils.py

```diff
@@ -366,8 +366,8 @@ def create_csv_for_github_benchmark_environment(github_benchmark_environment_csv
 
     device_info = json.dumps(
         {
-            "device_type": device_type,
-            "device_memory_size": device_memory_size,
+            "card_type": device_type,
```

Comment:
@tt-rkim is it possible to set device_type as n150/n300/t3000/galaxy or is that not exposed at the python level?

Reply:
That's part of #11134